bitnami/rust/1/debian-12/Dockerfile

```diff
@@ -8,14 +8,13 @@ ARG TARGETARCH
 
 LABEL com.vmware.cp.artifact.flavor="sha256:c50c90cfd9d12b445b011e6ad529f1ad3daea45c26d20b00732fae3cd71f6a83" \
       org.opencontainers.image.base.name="docker.io/bitnami/minideb:bookworm" \
-      org.opencontainers.image.created="2025-05-30T15:59:11Z" \
+      org.opencontainers.image.created="2025-06-26T23:19:17Z" \
```

Comment:
These are unrelated changes that were added to your PR. Could you please undo this commit and rebase the PR